proposals/all_proposals.md

```diff
@@ -0,0 +1,658 @@
+<lynX> DiEM is not in government, therefore it should not confuse electorate with "preliminary" steps. It should promote the long-term vision for fair technology that doesn't suck. Proprietary technology should not be legal for any societally critical applications.
+
+Data Protection and Privacy
+
+As pointed out above: The only real protection is with free/open source software. When it comes to the evasion of our privacy there are 2 major factors here:1.) spying by governments via there secret service organisations.2.) private data collection by corporations.As for 2.: The main motivation for private data collection is advertising.Advertising in itself is harmful to our society. It creates artificial demands. Its only product is the creation of discontent. People wanting more and not being happy with what they have. As the ecologic footprint of this is destroying our environment we need to cut down on the logic that forces constant growth anyway. Thus we should either put high taxes on advertising, or outright outlaw it. Also cutting down on trademark protection should remove incentives for advertising. – 
+
+<lynX> Outlawing advertising is a worthwhile thought. It is rarely ever used to inform us of the existence of some new product and most of the time manipulating us, especially everyone below 20 years of age or so (when advertising actually works best — check research on the subject). This is beyond the scope of our pillar, but we could certainly forbid all kinds of surveillance-based and individualised advertisement built on psychological manipulation by disabling certain features of HTTP and Javascript *by law*. No web browser is allowed to support them. No web server in Europe must try to employ them.
+
+A second motivation for private data-collection is the monetization of digital content. In order to charge money for some content, the content needs to be behind a pay-wall with personalized login. While in the old days, TV was broadcast and then could be received anonymously on a TV or Radio set. We should try to get back to this model: Cutting down on the Copyright protection so that content can be freely shared on the Internet without the need to have personalized logins. Content creators could be rewarded by anonymous statistical methods that measure of popularity of content. Also we should strive for public sponsoring of quality content.
```

Comment:
nteresting thought - gets at tradeoff between data protection for content consumers vs utility for value attribution to content creators - kate